build.gradle

```diff
@@ -152,94 +153,83 @@ testing {
-                   url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
-                   distribution = "repo"
-               }
-           }
+    pom {
+       description = project.description
+       name = project.name
+       url = "https://github.com/harness/ff-java-server-sdk"
 
-           scm {
-               developerConnection = "scm:git:ssh://https://github.com/harness/ff-java-server-sdk.git"
-               connection = "scm:git:ssh://https://github.com/harness/ff-java-server-sdk.git"
-               url = "https://github.com/drone/ff-java-server-sdk"
+       licenses {
+           license {
+               name = "Apache License, Version 2.0"
+               url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+               distribution = "repo"
            }
+       }
 
-           developers {
-               developer {
-                   id = "andybharness"
-                   name = "Andrew Bell"
-                   email = "andrew.bell@harness.io"
-                   organization = "Harness Inc"
-                   organizationUrl = "https://www.harness.io/"
-               }
-               developer {
-                   id = "davejohnston"
-                   name = "Dave Johnston"
-                   email = "dave.johnston@harness.io"
-                   organization = "Harness Inc"
-                   organizationUrl = "https://www.harness.io/"
-               }
-               developer {
-                   id = "enver-bisevac"
-                   name = "Enver Bisevac"
-                   email = "enver.bisevac@harness.io"
-                   organization = "Harness Inc"
-                   organizationUrl = "https://www.harness.io/"
-               }
-               developer {
-                   id = "rushabh-harness"
-                   name = "Rushabh Shah"
-                   email = "rushabh@harness.io"
-                   organization = "Harness Inc"
-                   organizationUrl = "https://www.harness.io/"
-               }
-               developer {
-                   id = "hannah-tang"
-                   name = "Hannah Tang"
-                   email = "hannah.tang@harness.io"
-                   organization = "Harness Inc"
-                   organizationUrl = "https://www.harness.io/"
-               }
-               developer {
-                   id = "subiradhikari"
-                   name = "Subir Adhikari"
-                   email = "subir.adhikari@harness.io"
-                   organization = "Harness Inc"
-                   organizationUrl = "https://www.harness.io/"
-               }
-           }
+       scm {
+           developerConnection = "scm:git:ssh://https://github.com/harness/ff-java-server-sdk.git"
+           connection = "scm:git:ssh://https://github.com/harness/ff-java-server-sdk.git"
+           url = "https://github.com/drone/ff-java-server-sdk"
        }
-    }
-}
 
-signing {
-    required { gradle.taskGraph.hasTask("publish") }
+       developers {
```

Comment:
Do these do anything? Should we change them to the current team and remove the legacy folks like dodgy @davejohnston

Reply:
gone, but not forgotten.